src/Homepage.js

```diff
@@ -0,0 +1,466 @@
+import React, { useState, Suspense, useEffect } from "react";
+import Grid from "@material-ui/core/Grid";
+import Tag from "./Tag";
+import keyBy from "lodash.keyby";
+
+import Flights from "./Flights";
+import StateGraph from "./StateGraph";
+import MbMap from "./ConfirmedMap";
+import GoogleMap from "./GoogleMap";
+
+import uuid from "react-uuid";
+import CanvasJSReact from "./assets/canvasjs.react";
+import provinces from "./data/area";
+import stateData from "./data/state";
+import testedCases from "./data/testedCases";
+import flights from "./data/flight";
+import country from "./data/country";
+import all from "./data/overall";
+import stateCaseData from "./data/stateCaseData";
+
+const provincesByName = keyBy(provinces, "name");
+
+export default function HomePage({
+  province,
+  overall,
+  myData,
+  area,
+  data,
+  setProvince,
+  gspace
+}) {
+  return (
+    <Grid container spacing={gspace} justify="center" wrap="wrap">
+      <Grid item xs={12} sm={12} md={10} lg={6} xl={4}>
+        <Stat
+          {...{ ...all, ...overall }}
+          name={province && province.name}
+          data={myData}
+          countryData={country}
+        />
+        <div className="card">
+          <h2>
+            Cases by State {province ? `· ${province.name}` : false}
+            {province ? (
+              <small onClick={() => setProvince(null)}>Return</small>
+            ) : null}
+          </h2>
+          <Suspense fallback={<div className="loading">Loading...</div>}>
+            <GoogleMap
+              province={province}
+              data={data}
+              onClick={name => {
+                const p = provincesByName[name];
+                if (p) {
+                  setProvince(p);
+                }
+              }}
+              newData={myData}
+            />
+            {/*{*/}
+            {/*province ? false :*/}
```

Comment:
remove?

Reply:
My branch was a bit out of date, so it remained some old files that's already deleted from the latest dev branch. The merge from Dev didn't remove such files from my branch. I have removed them manually.